apps/SageAccountWeb/src/style/_AccountSetting.scss

```diff
@@ -6,29 +6,25 @@
 
 .account-setting-panel {
   margin-bottom: 20px;
-  @media (max-width: 768px) {
-    &.nav-panel {
+  &.nav-panel {
+    @media (max-width: 768px) {
       display: none;
     }
-  }
-  &.nav-panel {
-    min-width: 200px;
     margin-right: 60px;
+    white-space: nowrap;
     box-shadow: 0px 8px 4px rgba(0, 0, 0, 0.025),
       0px 4px 2px rgba(0, 0, 0, 0.025), 0px 2px 1px rgba(0, 0, 0, 0.025);
-    border-radius: 3px;
     position: -webkit-sticky; /* for Safari */
     position: sticky;
     top: 60px;
     align-self: flex-start;
-    li {
-      background-color: #fff;
-      font-size: 14px;
+    padding-top: 10px;
+    padding-bottom: 20px;
```

Comment:
Match padding in [Figma](https://www.figma.com/file/TOcmJ2QHr51v8Huv3wIec9/Sage-Registration-App?node-id=1755-45706&t=exIPmwrsFpKOL1pJ-4) more closely